tft.py

```diff
@@ -116,30 +116,68 @@ def kill_process(process_executable: str, force: bool = True) -> subprocess.Comp
     )
 
 
+def launch_league_client(deceive_config: bool) -> None:
+    """
+    Launches the client
+
+    Args:
+        deceive_config: Whether to open the client through Riot or through Deceive
+    """
+    if deceive_config:
+        location = config.get_install_location_deceive()
+        if location is None:
+            logger.warning("Deceive is enabled in config.yaml, but no path was given! The bot will attempt to find the .exe on its own")    # pylint: disable=line-too-long
+            location = system_helpers.determine_deceive_install_location()
+            if location is None:
+                logger.error("Could not determine Deceive location. Please manually add the location in output/config.yaml")    # pylint: disable=line-too-long

```

Comment:
These could be split up using a multiline string, which would resolve the pylint errors 😄

Reply:
Went ahead and did that. I managed to squeeze line 132 into the limit without splitting it by replacing `"add the location"` with `"add the path"`. Could've tried that earlier...